NitroxServer/GameLogic/PlayerManager.cs

```diff
@@ -135,6 +140,12 @@ public void PlayerDisconnected(NitroxConnection connection)
                 }
 
                 assetsByConnection.Remove(connection);
+
+                if(ConnectedPlayers().Count == 0)
```

Comment:
Minor: No space after if statement; apply formatting to this file.

Reply:
Ty I missed that. I thought our editor config picked this up?

Reply:
Yes it should. If the code style is wrong then we should fix that.